File: CursoPOOuber/Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
    Map<String, ArrayList<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String licencia, Account driver, Map<String, 
    ArrayList<String, Integer>> typeCarAccepted, 
    ArrayList<String> seatsMaterial){
        super(licencia, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
        
    }
}

File: CursoPOOuber/Python/uberblack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, licencia, driver, typeCarAccepted, seatsMaterial):
        super.__init__(licencia, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial